README.md

```diff
@@ -47,17 +47,16 @@ We have a dedicated stream for this project [#mentorship-system](https://anitab-
 3. Fork and clone the [mentorship-flutter](https://github.com/anitab-org/mentorship-flutter) repository to your local machine.
 4. Open the project on your IDE. Don't be shocked if you see error warnings.
 5. Open the terminal and run `flutter channel beta` to switch from stable channel to beta channel of flutter. The beta channel is the one we are using for this project.
```

Comment:
This implicates we get the errors in IDE due to channel, can you shift flutter channel beta to step 4 and open IDE to step 5? Just to be safe, don't want to confuse others.

Reply:
Done

Reply:
Danke

Reply:
Bitte sehr